### ISSUE TITLE
feature: allow uploading tests that can be parameterized by TestProject

### ISSUE THREAD

**Issue by s-glatshtein** [testproject-io/csharp-opensdk#181]
Recently Java SDK added the ability to upload tests that can be parameterized by TestProject platform. The same should be done for C# SDK.

**Comment by Chrys123** [comment]
Yes I have this problem with C sharp coded tests upload, please fix this asap
